R/utils.R

```diff
@@ -465,7 +468,8 @@ average_biological_replicates_dt <- function(
   
   average_fields <- setdiff(names(Filter(is.numeric, data)), c(unlist(pidfs), var, iso_cols))
   geometric_average_fields <- intersect(geometric_average_fields, names(dt))
-  group_by <- setdiff(names(data), c(average_fields, var, id_cols, "fit_type", "Fit Type"))
+  fit_type_average_fields <- intersect(fit_type_average_fields, names(dt))
```

Comment:
Could you add some tests?

(so fit_type is also a part of definition of `unique record`, right?)

Reply:
Yes, in the sense that it belongs to the group of columns that should be skipped when averaging duplicates.

Reply:
done in f053c6bcb07c592848e59a619e2bef80a8cbade4.

Reply:
✔️